xbmc/pvr/addons/PVRClients.h

```diff
@@ -155,12 +155,21 @@ namespace PVR
     bool HasConnectedClients(void) const;
 
     /*!
-     * @brief Get the friendly name for the client with the given id.
+     * @brief Get the (friendly) name for the client with the given id.
      * @param iClientId The id of the client.
-     * @param strName The friendly name of the client or an empty string when it wasn't found.
+     * @param strName The (friendly) name of the client or an empty string when it wasn't found.
+     * @param bFriendly return friendly vs. plain addon name
      * @return True if the client was found, false otherwise.
      */
-    bool GetClientName(int iClientId, std::string &strName) const;
+    bool GetClientName(int iClientId, std::string &strName, bool bFriendly = true) const;
```

Comment:
It's more the other way around. Original implemenation of CPVRClients::GetClientName() used always CPVRClient::GetFriendlyName(), but I needed just the plain addon name (CAddon::Name) for the eventlog entries. Thus, every existing call to CPVRClient::GetClientName actually uses "friendly name".